java/compaction/compaction-core/src/test/java/sleeper/compaction/testutils/InMemoryCompactionJobStatusStoreTest.java

```diff
@@ -443,6 +468,7 @@ void shouldTrackJobStartedTwiceThenFinishedTwiceFromSameTask() {
             store.jobStarted(compactionJobStarted(job, startedTime1).taskId(taskId).jobRunId(runId1).build());
             store.jobStarted(compactionJobStarted(job, startedTime2).taskId(taskId).jobRunId(runId2).build());
             store.jobFinished(compactionJobFinished(job, summary1).taskId(taskId).jobRunId(runId1).build());
+            store.jobCommitted(compactionJobCommitted(job, committedTime1).taskId(taskId).jobRunId(runId1).build());
```

Comment:
This is a bit different from the actual flow now. I think this test was written when the state store committer added a job finished event, but it adds a committed event instead now. It would probably be clearer if we put the finished event for job 1 before the started event for job 2.